projects/saas-ui/src/app/on-boarding/on-boarding-service.ts

```diff
@@ -0,0 +1,156 @@
+import {HttpHeaders, HttpParams} from '@angular/common/http';
+import {Injectable} from '@angular/core';
+import {
+  AnyAdapter,
+  AnyObject,
+  ApiService,
+  Fields,
+  Inclusion,
+  Where,
+} from '@project-lib/core/api';
+import {AuthTokenSkipHeader} from '@project-lib/core/constants';
+import {Observable, Subject, filter, map, of} from 'rxjs';
+import {
+  VerifyEmailCommand,
+  GetLeadByIdCommand,
+  AddTenantFromLeadCommand,
+  GetPlanCommand,
+  AddLeadCommand,
+  GetLeadListCommand,
+} from './commands';
+import {ValidateLeadAdapter, GetPlanAdapter} from './adapters';
+import {Lead, Tenant, Plan} from './models';
+
+import {GetBillingDetails} from '../main/lead-list/commands/get-billing.command';
+interface BackendFilter<MT extends object = AnyObject> {
+  where?: Where<MT>;
+  fields?: Fields<MT> | (keyof MT)[];
+  order?: string[];
+  limit?: number;
+  skip?: number;
+  offset?: number;
+  include?: Inclusion[];
+}
+@Injectable({
+  providedIn: 'root',
+})
+export class OnBoardingService {
+  private readonly authTokenSkipHeader = new HttpHeaders().set(
+    AuthTokenSkipHeader,
+    '',
+  );
+
+  private validateLead$ = new Subject<Lead>();
+  constructor(
+    private readonly apiService: ApiService,
+    private readonly anyAdapter: AnyAdapter,
+    private readonly validateLeadAdapter: ValidateLeadAdapter,
+    private readonly getPlanAdapter: GetPlanAdapter,
+  ) {}
+
+  getValidateLead(): Subject<Lead> {
+    return this.validateLead$;
+  }
+  setValidateLead(lead: Lead) {
+    this.validateLead$.next(lead);
+  }
```

Comment:
where is this used, cant find its reference

Reply:
also check other unused code and remove

Reply:
removed